target/linux/ath79/image/common-mikrotik.mk

```diff
@@ -1,8 +1,10 @@
 define Device/mikrotik
 	DEVICE_VENDOR := MikroTik
+	LOADER_TYPE := elf
 	KERNEL_NAME := vmlinuz
 	KERNEL := kernel-bin | append-dtb-elf
-	KERNEL_INITRAMFS := kernel-bin | append-dtb-elf
+	KERNEL_INITRAMFS_NAME := vmlinux
```

Comment:
Guessing this should be vmlinux-initramfs

Reply:
Indeed:
```
6217344 openwrt-ath79-mikrotik-mikrotik_routerboard-map-2nd-initramfs-kernel.bin
```
Now the size is back to "normal" and the resulting image booted with its initramfs.